.gitignore

```diff
@@ -36,3 +36,5 @@ yarn-error.log*
 next-env.d.ts
 
 .env
+
+.sentryclirc
```

Comment:
shouldn't be here

Reply:
Sorry, it was in my local so I put it in .gitignore manually. will remove it.